### PR TITLE
fix(modal): align with the guidelines

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

Modal are not align with guidelines

**What is the chosen solution to this problem?**

http://1156.talend.surge.sh/theme/#dialogs

* Auto scroll content if it overflows

# Header

* $white background color
* text align to left
* add border bottom (already exists)

# content

* bg color to $whild-sand

# footer

* add border-top
* do not add the shadow effect

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/example/index.html [hunk 1]
```diff
@@ -1745,6 +1746,10 @@ <h4 class="modal-title">Source Code</h4>
     <script src="//code.jquery.com/jquery-1.10.2.min.js"></script>
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
     <script>
+      function add_content(cls) {
```
Comment (frassinier) [inline packages/theme/example/index.html]: Should be in IIF :)

File: packages/theme/src/theme/_modals.scss [hunk 2]
```diff
@@ -1,36 +1,40 @@
...
+	}
 	&-header {
 		border-top-left-radius: $border-radius-base;
 		border-top-right-radius: $border-radius-base;
 		background-color: $modal-header-bg;
+		border-color: $concrete;
 
 		.modal-title {
-			font-weight: bold;
+			font-weight: 400;
```
Comment (frassinier) [inline packages/theme/src/theme/_modals.scss]: default value, should be removed

File: packages/theme/src/theme/_modals.scss [hunk 3]
```diff
@@ -1,36 +1,40 @@
...
+			font-size: 16px;
```
Comment (frassinier) [inline packages/theme/src/theme/_modals.scss]: There is no existing variable for that font-size? 🤔

File: packages/theme/src/theme/_modals.scss [hunk 4]
```diff
@@ -1,36 +1,41 @@
...
 	display: flex;
 	flex-direction: column;
 }
 
 .modal {
-
+	&-content {
+		display: flex;
+		flex-direction: column;
+		border: none; // it keeps the border-radius
```
Comment (jsomsanith) [inline packages/theme/src/theme/_modals.scss]: The comment is not necessary

File: packages/theme/src/theme/_modals.scss (hunk 3 above)
Comment (jmfrancois) [reply packages/theme/src/theme/_modals.scss]: $form-input-font-size: 16px !default; :/

Comment (frassinier) [reply packages/theme/src/theme/_modals.scss]: ok forget it :(

File: packages/theme/src/theme/_modals.scss [hunk 5]
```diff
@@ -1,36 +1,41 @@
-
 .modal-dialog {
 	max-height: 80vh;
 	display: flex;
 	flex-direction: column;
 }
 
 .modal {
-
+	&-content {
```
Comment (frassinier) [inline packages/theme/src/theme/_modals.scss]: May it be placed after `&-header`? (nice to have)

File: packages/theme/example/index.html (hunk 1 above)
Comment (jmfrancois) [reply packages/theme/example/index.html]: I need it as a global

File: packages/theme/src/theme/_modals.scss (hunk 5 above)
Comment (jmfrancois) [reply packages/theme/src/theme/_modals.scss]: no, because header is inside content

**Pull request merged**